### ISSUE TITLE
v0.2.0 plugin errors

### ISSUE THREAD

**Issue by jr0dd** [ori-edge/k8s_gateway#70]
Just updated and the logs are filled with plugin errors and ingresses do not resolve. Rolling back to previous version fixes.

logs
```
[INFO] plugin/k8s_gateway: Building k8s_gateway controller
[INFO] plugin/k8s_gateway: GatewayAPI CRDs are not found. Not syncing GatewayAPI resources.
[INFO] plugin/k8s_gateway: Starting k8s_gateway controller
.:1053
[INFO] 127.0.0.1:57272 - 33036 "HINFO IN 7105407453865849624.6631760172766430674. udp 57 false 512" NOERROR - 0 0.000028511s
[ERROR] plugin/errors: 2 7105407453865849624.6631760172766430674. HINFO: plugin/loop: no next plugin found
[INFO] plugin/reload: Running configuration MD5 = 4de2454b8067b23156eec571e008b5b5
CoreDNS-1.8.3
linux/amd64, go1.16.13, 1ef6d2c
[INFO] plugin/k8s_gateway: Waiting for controllers to sync
[INFO] plugin/k8s_gateway: Synced all required resources
[INFO] 10.10.0.1:33378 - 488 "NS IN . udp 17 false 512" NOERROR - 0 0.000037287s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[INFO] 10.10.0.1:34698 - 59050 "NS IN . udp 17 false 512" NOERROR - 0 0.000009432s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[INFO] 10.10.0.1:48821 - 9460 "NS IN . udp 17 false 512" NOERROR - 0 0.00000736s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[INFO] 10.10.0.1:46148 - 29352 "NS IN . udp 17 false 512" NOERROR - 0 0.000007447s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[INFO] 10.10.0.1:45872 - 15683 "NS IN . udp 17 false 512" NOERROR - 0 0.000006294s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[ERROR] Recovered from panic in server: "dns://:1053"
[INFO] 10.10.0.1:38670 - 60566 "NS IN . udp 17 false 512" NOERROR - 0 0.000076398s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[INFO] 10.10.0.1:46664 - 55158 "NS IN . udp 17 false 512" NOERROR - 0 0.000037465s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[INFO] 10.10.0.1:41221 - 61769 "NS IN . udp 17 false 512" NOERROR - 0 0.000037325s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[INFO] 10.10.0.1:58622 - 1452 "NS IN . udp 17 false 512" NOERROR - 0 0.000032213s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[INFO] 10.10.0.1:52451 - 4375 "NS IN . udp 17 false 512" NOERROR - 0 0.000031436s
[ERROR] plugin/errors: 2 . NS: plugin/loop: no next plugin found
[ERROR] Recovered from panic in server: "dns://:1053"
[ERROR] Recovered from panic in server: "dns://:1053"
[ERROR] Recovered from panic in server: "dns://:1053"
```

helm values
```
nameOverride: k8s-gateway
domain: domain.tld
ttl: 1
service:
  type: LoadBalancer
  port: 53
  externalTrafficPolicy: Local
  loadBalancerIP: 10.10.2.2
```

**Comment by networkop** [comment]
@jr0dd this is fixed in master and in will be in 0.2.1 later today

**Comment by jr0dd** [comment]
> @jr0dd this is fixed in master and in will be in 0.2.1 later today.
> thanks for reporting

Woot! That fixed it up. thanks!
